### ISSUE TITLE
More precise game time

### ISSUE THREAD

**Issue by Jannify** [urwrstkn8mare/gog-riot-integration#7]
Is it possible to get the current playtime on installing with this [site](https://wol.gg/)? I didn't found any request that returns the time because it's integrated in the HTML but maybe there is a way besides downloading the HTML document and get the time out of it...

**Comment by urwrstkn8mare** [comment]
Thanks for pointing it out. From what I've seen online about this (I've looked at it before) if doesn't get the actual gametime but gets the number of games and estimates. That's why you'll see drastically different game times on similar sites. But I could implement if is it would still be useful. Let me know if you would still want it and I'll add that to my list of things to work on.

**Comment by Jannify** [comment]
I think it would be pretty neat to update the game time on installation/authentication (if not already set)). So GOG continues the accurate counting not from zero.

**Comment by urwrstkn8mare** [comment]
Oh yeah that does seem like a pretty good idea. I'll try to get that done soon. Thanks

**Comment by innarik** [comment]
I agree with the Jannify idea, pls.

**Comment by tippfehlr** [comment]
Wouldn't it be easier if you (for now) just add a configuration field or something in that direction where you can set your playtime?

**Comment by urwrstkn8mare** [comment]
Yeah but I just don't have the time. If someone does it and opens a pull request I'll happily quickly review and accept. Sorry
